src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
+  let totalCost = days * dayCost;
+
+  if (days >= 3 && days < 7) {
+    totalCost = totalCost - 20;
```

Comment:
Extract the numbers 3, 7, 20, and 50 into appropriately named constants to avoid hardcoding and make the code more readable.